sdk-api-src/content/winternl/nf-winternl-ntnotifychangemultiplekeys.md

```diff
@@ -62,23 +62,23 @@ A handle to an open key. The handle must be opened with the <b>KEY_NOTIFY</b> ac
 
 ### -param Count [in, optional]
 
-The number of subkeys under the key specified by the <i>MasterKeyHandle</i> parameter. This parameter must be 1.
+The number of keys objects provided in the <i>SubordinateObjects</i> parameter. This parameter must be 1.
 
 ### -param SubordinateObjects [in, optional]
 
-Pointer to an array of <a href="/windows-hardware/drivers/ddi/content/wudfwdm/ns-wudfwdm-_object_attributes">OBJECT_ATTRIBUTES</a> structures, one for each subkey.   This array can contain one <b>OBJECT_ATTRIBUTES</b> structure.
+Pointer to an array of <a href="/windows-hardware/drivers/ddi/content/wudfwdm/ns-wudfwdm-_object_attributes">OBJECT_ATTRIBUTES</a> structures, one for each key.   This array can contain one <b>OBJECT_ATTRIBUTES</b> structure and must not be a key in the same hive as the <i>MasterKeyHandle</i> key.
```

Comment:
If these aren't in the same hive as the master root key, how then could they be "subkeys" of a given registry key, as the summary description of this function claims they are?
What then would be the usefulness of the function altogether if this weren't the case?

Reply:
Being a subkey isn't a requirement, I tested the API on \Registry\User\.Default\TestKey while the master key was \Registry\Machine\Software\TestKey and it worked. While being a subkey is not a requirement, it's possible to load a hive in a non-root key so that way a key would have a subkey from another hive. As far as I know that's how BCD and SAM database are loaded in to the registry, they're not a root key like HKLM but a subkey of a parent key, but they're from a separate hive compared to their parent.